Public/Src/Cache/ContentStore/Distributed/NuCache/CheckpointManager.cs

```diff
@@ -90,17 +93,24 @@ public Task<BoolResult> CreateCheckpointAsync(OperationContext context, EventSeq
                         // Creating a working temporary directory
                         using (new DisposableDirectory(_fileSystem, _checkpointStagingDirectory))
                         {
+                            // NOTE(jubayard): this needs to be done previous to checkpointing, because we always
+                            // fetch the latest version's size in this way. This implies there may be some difference
+                            // between the reported value and the actual size on disk: updates will get in in-between.
+                            // The better alternative is to actually open the checkpoint and ask, but it seems like too
+                            // much.
+                            checkpointSize = _database.GetContentDatabaseSizeBytes().ThrowIfFailure();

```

Comment:
I don't think we should fail if we won't get the size. Maybe just print -1 as the size in this case?